content/1_docs/3_reference/4_objects/0_page/0_has-next-listed/method.txt

```diff
@@ -8,10 +8,11 @@ echo $page->hasNextListed(); //will echo 1 if true, nothing if false.
 ?>
 ```
 
-### Using sort field
+### With collection as argument
 
 ```php
 <?php
-echo $page->hasNextListed('date', 'desc'); //will echo 1 if true, nothing if false.
+$collection = $page->siblings(false)->listed()->sortBy('date', 'desc');
```

Comment:
Does this work with `->siblings(false)`? The `next`/`prev` methods need to find the position of the current `$page` in the collection to move from there. So if the `$page` itself is not part of the collection, I wonder how it will work. But I haven't tested this.

Unrelated change: The `will echo 1 if true, nothing if false.` comments are quite confusing. The `has*` methods actually return a boolean, so it would be better if the examples used `var_dump()`, which will clearly display the result and will make the comment unnecessary.

Reply:
Good question, I'll double-check, but I think you are absolutely right and I just didn't think properly.